testnet/stacks-node/src/run_loop/neon.rs

```diff
@@ -1092,6 +1092,16 @@ impl RunLoop {
                 }
             };
 
+            // print burnchain sync percentage
+            if ibd {
+                let percent: f64 =
+                    burnchain_tip.block_snapshot.block_height as f64 / remote_chain_height as f64;
+                debug!("Burnchain sync percentage";
+                       "percent" => %percent,
+                       "local_tip_height" => %burnchain_tip.block_snapshot.block_height,
+                       "remote_tip_height" => %remote_chain_height);
+            }
+
```

Comment:
Can this be incorporated into the `debug!` message on line `1113`, e.g.:

```rust
            debug!(
                "Runloop: Downloading burnchain blocks up to reward cycle #{} (height {})",
                burnchain_config
                    .block_height_to_reward_cycle(target_burnchain_block_height)
                    .expect("FATAL: target burnchain block height does not have a reward cycle"),
                target_burnchain_block_height;
                "total_burn_sync_percent" => %percent,
                "local_burn_height" => burnchain_tip.block_snapshot.block_height,
                "remote_tip_height" => remote_chain_height
            );
```

That way, this change doesn't add any additional log lines.